src/components/Menus/Sidebar/Sidebar.jsx

```diff
@@ -57,10 +57,14 @@ const Sidebar = () => {
       position: "bottom",
       panel: InfoPanel,
     },
-  ];
+  ].filter((option) => options.includes(option.name));
```

Comment:
💪 nice and expandable!! It might be worth pulling this out at some stage into `SidebarPanels.js` or something similar to future proof it but one to mull over